### ISSUE TITLE
Moving to dotnet 6

### ISSUE THREAD

**Issue by rakamaric** [boogie-org/boogie#563]
I vaguely remember seeing the discussion about moving to dotnet 6 somewhere, but not I can't find it any more. Sorry of I am duplicating this.
In short, what we learned past week is that it seems that dotnet 5 doesn't run natively on the new M1 mac machines. This leads to some libraries missing (or being incompatible) when we tried to run Boogie. This seems to be what we are seeing: https://github.com/dotnet/installer/issues/12187#issuecomment-930349190

Long story short, it would be nice to move Boogie to dotnet 6. We tried that and everything seems to be working fine.

**Comment by keyboardDrummer** [comment]
Boogie is already buildings against .NET 6: https://github.com/boogie-org/boogie/blob/master/Source/Directory.Build.props#L6

Can we close this issue?

**Comment by rakamaric** [comment]
Yep, I'll close it. It was Corral in the end and not Boogie. Sorry about mixing those up.
